cmd/proxyplane/opv-proxyplane-http.example.json

```diff
@@ -0,0 +1,69 @@
+{
+  "version": 2,
+  "name": "OPV Proxy Plane",
+  "port": 28001,
+  "debug": false,
+  "host": [
+    "127.0.0.1"
+  ],
+  "cache_ttl": 3600,
+  "timeout": "3s",
+  "endpoints": [
+    {
+      "endpoint": "/tokenize",
+      "method": "POST",
+      "backend": [
+        {
+          "host": [
+            "https://httpbin.org"
+          ],
+          "url_pattern": "/post",
```

Comment:
what's this?

Reply:
the upstream is defined as a combination of `host` and `url_pattern`. `url_pattern` can template the incoming requests' path - an advanced feature like https://www.krakend.io/docs/endpoints/parameter-forwarding/#mandatory-query-string-parameters